TodoApp/TodoApp.Api/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("TodoApp.Api")]
+[assembly: AssemblyDescription("")]
```

Comment:
If you want, you can specify info for the main assembly. It is best practice when you have public library/nuget (not your case, but having a habit helps).